CHANGES.rst

```diff
@@ -52,7 +52,7 @@
     - Added a function ``check_spectrograph()`` (currently only defined for LRIS),
       that checks (during ``pypeit_setup``) if the selected spectrograph is the
       corrected one for the data used. 
-
+- Introduced PCA method for telluric corrections
```

Comment:
Subsequent to this PR, the use of CHANGES has been deprecated, so I'll make a quick note of that here...